src/site/stages/build/drupal/graphql/landingPage.graphql.js

```diff
@@ -24,10 +24,16 @@ module.exports = `
       entity {
         ...listOfLinkTeasers
       }
-    }    
-    fieldSupportServices {          
-      ...on FieldNodeFieldSupportServices {            
-        entity {      
+    }
+    fieldLinks {
```

Comment:
@ethanteague this probably needs to be feature flagged if it is not on prod yet!